test/symbol-test.js

```diff
@@ -48,8 +48,8 @@ tape("symbol.type(symbolHexagon) generates a polygon with the specified size", f
 
 tape("symbol.type(symbolHexagon) generates the expected path", function(test) {
   var s = shape.symbol().type(extra.symbolHexagon).size(function(d) { return d; });
-  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0L0,0Z");
-  test.pathEqual(s(10), "M1.699044,0.980944L1.699044,0.980944L0,1.961887L-1.699044,0.980944L-1.699044,-0.980944L0,-1.961887L1.699044,-0.980944Z");
+  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0Z");
+  test.pathEqual(s(10), "M1.699044,0.980944L0,1.961887L-1.699044,0.980944L-1.699044,-0.980944L0,-1.961887L1.699044,-0.980944Z");
```

Comment:
<img width="442" alt="1__vim" src="https://cloud.githubusercontent.com/assets/529247/25559102/6c99239c-2d3d-11e7-998e-ee45601bd38a.png">
The first point was used two times.